modules/virt-exposing-pci-device-in-cluster-cli.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-exposing-pci-device-in-cluster-cli_{context}"]
+= Exposing PCI host devices in the cluster using the CLI
+
+To enable the availability of PCI host devices in the cluster, add details about the PCI devices to the `HyperConverged` custom resource (CR). The PCI devices are exposed and permitted to be used in the cluster.
+
+
+.Procedure
+. Edit the `HyperConverged` CR in YAML format using the default editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv -o yaml
+----
+
+. Add the PCI device information to the `spec.permittedHostDevices.pciHostDevices` array. For example:
++
+.Example configuration file
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+spec:
+  permittedHostDevices: <1>
+    pciHostDevices: <2>
+    - pciDeviceSelector: "10DE:1DB6" <3>
+      resourceName: "nvidia.com/GV100GL_Tesla_V100" <4>
+    - pciDeviceSelector: "10DE:1EB8"
+      resourceName: "nvidia.com/TU104GL_Tesla_T4"
+    - pciDeviceSelector: "8086:6F54"
+      resourceName: "intel.com/qat"
+      externalResourceProvider: true <5>
+...
+----
+<1> The host devices that are permitted to be used in the cluster.
+<2> The list of PCI devices available on the node.
+<3> The `vendor-ID` and the `device-ID` required to identify the PCI device.
+<4> The name of a PCI host device.
+<5> Optional field. When set to `true` indicates that the resource is provided by an external device plug-in. {VirtProductName} allows the usage of this device in the cluster but leaves the allocation and monitoring to an external device plug-in.
++
+[NOTE]
+====
+The above example snippet shows two PCI host devices that are named `nvidia.com/GV100GL_Tesla_V100` and `nvidia.com/TU104GL_Tesla_T4` added to the list of permitted host devices in the `HyperConverged` CR. These devices have been tested and verified to work with {VirtProductName}.
+====
+
+. Save your changes and exit the editor.
+
+.Verification
+* Verify that the PCI host devices were added to the node by running the following command. The example output shows that there is one device each associated with the `nvidia.com/GV100GL_Tesla_V100`, `nvidia.com/TU104GL_Tesla_T4`, and `intel.com/qat` resource names.
++
+[source,terminal]
+----
+$ oc describe node <node1>
```

Comment:
I think that instead of `<node1>` this should be something like `<node_name>`. If you want to use `node1` as an example name I would take the `<>` out.